packages/StyleCop.MSBuild.4.7.54.0/tools/Settings.StyleCop

```diff
@@ -0,0 +1,863 @@
+<StyleCopSettings Version="105">

```

Comment:
Could be, but as the StyleCop editor is not integrated into VS you separate the tool from the file, but would make the fact that StyleCop is used more visible. So there are pros and cons.

The way it is now makes it easy to add StyleCop to new C# projects via NuGet. I have not investigated if moving the Settings file makes it more difficult to add StyleCop to new projects.

There is also the issue with the future of StyleCop. It has been dormant for quite a while and still does not support C#6